frontend/src/components/form/components/Upload.vue

```diff
@@ -83,34 +83,45 @@ export default {
         parentObject: Object,
     },
     data() {
-        let iu = (typeof(this.currentlyUrl) === "boolean") ? this.currentlyUrl : null;
-        iu = (typeof(this.currentlyUrl) === "string") ? (this.currentlyUrl.toLowerCase()==='true') : iu;
-        iu = ( (iu === null) && (typeof(this.isURL) !== "undefined") ) ? this.isUrl : iu;
-        iu = ( (iu === null) && (this.field.field_name === "url") ) ? (this.field.field_name === "url"): iu;
-
-        let v = (( (this.field.required) || (this.field.validators && this.field.validators.indexOf('conditional_required')!==-1) ) ? 'required' : '');
-        if (iu) {
-            v = {
-                url: {require_tld: true, require_host: true},
-                required: (v === "required")
-            }
+        let isURL;
+    
+        if (typeof(this.currentlyUrl) === "boolean") {
+            isURL = this.currentlyUrl;
+        } else {
+            isURL = (
+                typeof(this.currentlyUrl) === "string" &&
+                this.currentlyUrl.toLowerCase() === "true"
+            ) || this.field.field_name === "url";
         }
 
         return {
             val: this.value,
             fileVal: (this.upload) ? this.upload : null,
-            validate: v,
             allowURL: this.field.field_name === "url",
-            isURL: iu,
             scopeName: this.scope + '.' + this.name,
             api: null,
```

Comment:
What is the reasoning behind removing the validated functionality from the data section, and will removing it from the return break anything?

Reply:
It's removed from this section to fix the bug that was preventing files from being uploaded to existing resources; the issue with having this in the data section instead of the computed section is that the data section is not automatically refreshed when dependent fields change. We need it to be refreshed because if it is not, validation errors intended for URLs are thrown for file upload objects.